src/content/migrate/4.md

```diff
@@ -1,9 +1,18 @@
 ---
 title: To v4 from v3
+group: Migrate
```

Comment:
Yes @Kolhar730  the content looks pretty good to me, thanks a lot for taking care of this! There is one last oddity i have spotted, when you have added a group there is not a redundant header in side menu, please remove it and should be good. You can see it here in preview: https://deploy-preview-2834--webpackjsorg-netlify.netlify.com/migrate/

or on this screenshot
![Annotation 2019-07-31 212752](https://user-images.githubusercontent.com/10549495/62237742-367e4c80-b3da-11e9-9223-bd1e266f2cb8.png)

```suggestion
```